go-continuous-fuzz-chart/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
We can probably make this unconditionally create the svcaccount as the pod being spawned always relies on it existing.